README.md

```diff
@@ -11,8 +11,32 @@ the ability simulates the following PDUs:
  - [Baytech MRP27 PDU](https://github.com/openstack/virtualpdu/blob/master/virtualpdu/pdu/baytech_mrp27.py)
 
 
-Usage
------
+Quick start
+-----------
+
+### Using Docker ###
+
+#### From source ####
+
+        docker build -t virtualpdu-container .
```

Comment:
We should start those lines with only 4 spaces, it has a visual impact when rendered.